### ISSUE TITLE
Metamodels that are referenced by their relative workspace URI are silently ignored if they are not present in the Workspace

### ISSUE THREAD

**Issue by MishaStrittmatter** [kit-sdq-emf-refactor-fork/edu.kit.ipd.sdq.emf.refactor#6]
This issue is not to be resolved now, but later when there is enough time. For now, this issue just serves as documentation.

Elements of other metamodels in the workspace can be referenced as follows (in ecore/xmi):
eSuperTypes="../../de.uka.ipd.sdq.stoex/model/stoex.ecore#//RandomVariable"
If the metamodel is not in the workspace, but installed via a plugin in the platform (i.e., Eclipse), EMF notices this and uses the metamodel from the installed plugin.

However, it seems to me, that EMF Refactor cannot handle such situations. If such a metamodel is not in the workspace, but the platform, EMF Refactor reports less smells (compared to an analysis where the metamodel is in the workspace) but does not report an error.

In conclusion, if a metamodel that is referenced in the workspace is not found, an error should be reported and the analysis should be aborted.
